### PR TITLE
[FIX] can't load dapp with unknown network

### PR DESCRIPTION
**Pull request by tommasini**
**Description**
The problem was when we browse "totee.xyz" or "ebisusbay.xyz" and we are in the custom network it gives an EnsIpfsResolver error.

That error comes from the fact that we were verifying if the domain it's an ENS and actually is a top-level domain ending in '.xyz'.

**Proposed Solution**
The proposed solution it's that when the ens throws an error because doesn't have an ens for the custom chain id and ends in ".xyz" and ".test", we add it to the ensIgnoreList and return the full URL "https://ebisusbay.xyz"

Also changed the "substr" deprecated javascript method for "slice".

**Code Impact**
Low 

**Screenshots/Recordings**
Solution:
https://recordit.co/4zp4r2XzjG

**Test Cases**
Case1:
* Go to browser
* browse ebisusbay.xyz
* It redirects to https://app.ebisusbay.com/
* It loads without any alert/error

**Checklist**

* [x] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

**Issue**

Progresses #3391

### CODE REVIEW

File: app/components/Views/BrowserTab/index.js [hunk 1]
```diff
@@ -528,10 +528,16 @@ export const BrowserTab = (props) => {
 			} catch (err) {
 				// This is a TLD that might be a normal website
 				// For example .XYZ and might be more in the future
-				if (hostname.substr(-4) !== '.eth' && err.toString().indexOf('is not standard') !== -1) {
+
+				if (
```
Comment (tommasini) [inline app/components/Views/BrowserTab/index.js:532]: I don't know why it was not catching the XYZ, any thoughts?

File: app/components/Views/BrowserTab/index.js [hunk 2]
```diff
@@ -433,6 +433,18 @@ export const BrowserTab = (props) => {
     setTimeout(() => setShowPhishingModal(true), 1000);
   };
 
+  /**
+   * This function verify if the hostname is a TLD
+   * @param {string} hostname - Represents the hostname of the URL
+   * @param {*} error - Represents the error of handleIpfsContent
+   * @returns {Boolean} - True if its a TLD, false if it's not
+   */
+  const isTLD = (hostname, error) =>
```
[the remark this answers is not in the document]
Comment (tommasini) [reply app/components/Views/BrowserTab/index.js]: Done 🚀

**Review by sethkfman — APPROVED** [review]
Looks good to me

**Comment by chrisleewilcox** [comment]
Scenario 1: https://recordit.co/81O3dxxE0V
Scenario 2: https://recordit.co/O0NzoK8lzA
Scenario 3: https://recordit.co/YrW4Yn7ena

**Review by chrisleewilcox — APPROVED** [review]
LGTM.

**Pull request merged**